### DISCUSSION TITLE
Modal vs "inline" preview?

### DISCUSSION THREAD

**Discussion by ovizii** [karakeep-app/karakeep#121] [Q&A] [answered]
Not sure how to create a better title for this question, but I noticed when you bookmark using the Chrome extension, there is a preview link which looks like: http://192.168.192.2:3000/dashboard/preview/amd3pc2ry6lvu54m9unedri9 so I was wondering how to reach this link from Hoarder's GUI as in the GUI I can only see the "modal" preview? 

Let me try and explain with some screenshots:

![image](https://github.com/MohamedBassem/hoarder-app/assets/837902/7305fee6-2964-4412-8e54-01704ea276bd)

VS:

![image](https://github.com/MohamedBassem/hoarder-app/assets/837902/58b6d2b0-2c62-4391-b01a-7ec6ca1f3abc)

![image](https://github.com/MohamedBassem/hoarder-app/assets/837902/f4357a60-241b-414c-92c2-2ab4e99cd83a)

**Comment by MohamedBassem** [comment, marked as answer]
Lol. This is a very weirdly timed question :D I've literally just implemented this an hour ago (https://github.com/MohamedBassem/hoarder-app/commit/ec15d20abac65bfda6c8e6396ce9fc2061f5067a) and it'll be available in the next release. Basically, when you open the modal, the URL bar will point to the preview URL. If you open that URL directly (or refresh), you'll go the inline view.

Example:

<img width="1296" alt="Screenshot 2024-04-24 at 3 39 55 PM" src="https://github.com/MohamedBassem/hoarder-app/assets/2418637/1f30d5ba-0725-480c-b381-b936515cc87c">

**Reply by ovizii** [reply]
I wanted to ask this question for some time but forgot all about it. I remembered today because I noticed new features (tagging and adding to lists) in the extension and remembered to ask. :-)
